forte/data/data_store.py

```diff
@@ -593,6 +647,35 @@ def add_annotation_raw(
 
         return self._add_entry_raw(Annotation, type_name, entry)
 
+    def add_audio_annotation_raw(
+        self, type_name: str, begin: int, end: int, tid: Optional[int] = None
```

Comment:
For annotation-like entries, we can put `allow_duplicate` here. You may refer to https://github.com/asyml/forte/blob/e2ce0a4684c884068f88d4706184304cddedb1f7/forte/data/data_store.py#L556 to see how to incorporate `allow_duplicate` into this function.